libheif/heif.cc

```diff
@@ -33,7 +33,7 @@
 #include <set>
 #include <limits>
 
-#if defined(__EMSCRIPTEN__)
+#if defined(__EMSCRIPTEN__) && !defined(__EMSCRIPTEN_STANDALONE_WASM__)
```

Comment:
Note: `__EMSCRIPTEN_STANDALONE_WASM__` isn't something that Emscripten provides, I add this using `CMAKE_C_FLAGS` / `CMAKE_CXX_FLAGS`. Embind is not supported on non-web runtimes since it heavily relies on JS, so we do not want the Embind things to be injected in the standalone build.